test_scripts/API/Navigation/SendLocation/017_SendLocation_MandatoryOnly_Float.lua

```diff
@@ -0,0 +1,70 @@
+---------------------------------------------------------------------------------------------
+-- Requirements: https://github.com/smartdevicelink/sdl_requirements/blob/master/detailed_docs/TRS/embedded_navi/SendLocation_TRS.md
+--
+-- Requirement summary:
+-- In case mobile application sends valid SendLocation request, SDL must:
+-- 1) transfer Navigation.SendLocation to HMI
+-- 2) on getting Navigation.SendLocation ("SUCCESS") response from HMI, respond with (resultCode: SUCCESS, success:true)
+-- to mobile application.
+--
+-- Description:
+-- App sends SendLocation request with only mandatory parameters with float values.
+--
+-- Steps:
+-- mobile app requests SendLocation with mandatory parameters longitudeDegrees, latitudeDegrees with float values
+--
+-- Expected:
+-- SDL must:
+-- 1) transfer Navigation.SendLocation to HMI
+-- 2) on getting Navigation.SendLocation ("SUCCESS") response from HMI, respond with (resultCode: SUCCESS, success:true)
+-- to mobile application.
+---------------------------------------------------------------------------------------------------
+
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonSendLocation = require('test_scripts/API/Navigation/commonSendLocation')
+
+--[[ Local Variables ]]
+local request_params = {
+    longitudeDegrees = 1.1,
+    latitudeDegrees = 1.1
+}
+
+--[[ Local Functions ]]
+local function send_location(params, self)
+    local cid = self.mobileSession1:SendRPC("SendLocation", params)
+
+    params.appID = commonSendLocation.getHMIAppId()
+
+    EXPECT_HMICALL("Navigation.SendLocation", params)
+    :Do(function(_,data)
+        --hmi side: sending Navigation.SendLocation response
+        self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", {})
+    end)
+
+    self.mobileSession1:ExpectResponse(cid, { success = true, resultCode = "SUCCESS" })
+    :ValidIf (function(_,data)
+        if data.payload.info then
+            print("SDL sent redundant info parameter to mobile App ")
```

Comment:
@GetmanetsIrina , suggest to a little bit correct 
print("SDL sent redundant \"info\" parameter to mobile App")